spec/index.html

```diff
@@ -6814,12 +6818,16 @@ <h5>ENCODE_FOR_URI</h5>
           <section id="func-concat">
             <h5>CONCAT</h5>
             <pre class="prototype nohighlight"><span class="return">string literal</span>  <span class="operator">CONCAT</span>(<span class="type">string literal</span> <span>ltrl<sub>1</sub></span> ... <span class="type">string literal</span> <span>ltrl<sub>n</sub></span>)</pre>
-            <p>The <code>CONCAT</code> function corresponds to the XPath <a data-cite="XPATH-FUNCTIONS#func-concat">fn:concat</a> function. The function accepts string
-              literals as arguments.</p>
-            <p>The lexical form of the returned literal is obtained by concatenating the lexical
-              forms of its inputs. If all input literals are literals with identical language
-              tags, then the returned literal is a literal with the same language tag, in all
-              other cases, the returned literal is a literal with datatype <code>xsd:string</code>.</p>
+            <p>The <code>CONCAT</code> function returns a string literal following the 
+              <a href="#string-literal-return-type">string literal return type</a> rule.
+              The <a data-cite="RDF12-CONCEPTS#dfn-lexical-form">lexical form</a> of the
+              returned string literal is obtained by concatenating the lexical
```

Comment:
I notice that the cross-referenced section (i.e., Section 17.4.3.1.3 String Literal Return Type) does not actually state any rule for CONCAT. Instead, it only says that "The function CONCAT returns a string literal based on the details of all its arguments." So, when coming to that section, the reader is essentially referred back to this section here. This circular cross-referencing is somewhat contradictory. Therefore, I propose to simply remove the reference from this section here.
```suggestion
            <p>The <code>CONCAT</code> function returns a string literal such that
              the <a data-cite="RDF12-CONCEPTS#dfn-lexical-form">lexical form</a> of
              this literal is obtained by concatenating the lexical
```